### ISSUE TITLE
A11y_Windows Terminal_Open any Application_Color Contrast: While Opening any application which is not existing then the error message will be displayed and it is not meeting the color contrast ratio 4.5:1

### ISSUE THREAD

**Issue by SpandanaPaladi** [microsoft/terminal#5393]
#A11y_WindowsTerminal; #WindowsTerminal; #A11yMAS; #A11yTCS; #DesktopApp; #Win10; #C+E-H; #ColorContrast; #A11ySev1; #BenchMark; #MAS1.4.3

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shcha1%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847087032&sdata=uWx6cTEfazn%2BoCV3cL1iYB4KN%2BuXQ/MTlmOVec%2BZgKc%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Version: 0.10.781.0

**Repro steps:**

1. Hit the URL ""aka.ms/terminal"" and install ""Windows terminal (Preview)"" application
2. Open ""Windows terminal (Preview)"" application
3. Open the "Accessibility insights" tool to check color contrast ratio
4. Tab to ""Windows Powershell"" tab
5. Type any application name which is not existing(displays an error message)
6. Verify whether the color contrast ratio for the error message displayed is meeting 4.5:1 contrast ratio

**Actual Result:​**
The "Error" message which is displayed is not meeting  4.5:1 contrast ratio

**Expected Result:​**
The "Error" message which is displayed should meet 4.5:1 contrast ratio
​
**User Impact:​**
Low vision users will face difficulty in viewing the text if the text Error message fails to meet the contrast ratio 4,.5:1

**MAS Reference:​**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

![Bug_color contrast fails for error message](https://user-images.githubusercontent.com/63839131/79562628-03472f00-80c9-11ea-85ce-08ab34d81031.png)

**Comment by carlos-zamora** [comment]
Moving this one over to v1.0 milestone because we discussed offline that changing the color scheme to Campbell should suffice. (assuming Campbell provides a good color contrast)

**Comment by carlos-zamora** [comment]
#5432 is included in 0.11. Updating tag.

**Comment by Amulya-95** [comment]
Regressed the bug in latest environment. The issue is resolved, contrast ratio for "Error" message is greater than 4.5:1 ratio. Hence closing this bug. 

![Fixed Bug 5393_Color contrast ratio passes for error message](https://user-images.githubusercontent.com/51435266/81691327-41423380-947a-11ea-8b90-a274c2c8a09d.png)
